hack/make/release-deb

```diff
@@ -14,6 +14,7 @@ set -e
 #
 # ... and so on and so forth for the builds created by hack/make/build-deb
 
+source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
```

Comment:
Please ignore, `.detect-daemon-osarch` exports more variable than that. We should look at a better way, but if this solution work then let's move forward with it.